workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/hooks/useSetTouchedOnForms.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {FormikErrors, FormikHelpers, setNestedObjectValues} from 'formik';
```

Comment:
Do we have any in house standard for form/error handling? Feels odd to be importing an external library for error handling